components/dashboard/sections/accounting/AccountingCategoryForm.tsx

```diff
@@ -99,78 +136,46 @@ export function AccountingCategoryForm(props: AccountingCategoryFormProps) {
     },
   ];
 
+  const query = useQuery<AccountingCategoryFormQuery, AccountingCategoryFormQueryVariables>(
+    gql`
+      query AccountingCategoryForm($hostSlug: String!) {
+        host(slug: $hostSlug) {
+          id
+          type
+        }
+      }
+    `,
+    {
+      context: API_V2_CONTEXT,
+      variables: {
+        hostSlug: props.hostSlug,
+      },
+    },
+  );
+
+  if (query.loading) {
+    return <Loading />;
+  }
+
+  const isIndependentCollective = query.data.host.type === CollectiveType.COLLECTIVE;
```

Comment:
Same here, adding a full query while we could have just passed the prop down seems like unnecessary complexity. Especially since the query is not shared between components, we could end up in a situation where a developer changes one but not the others and we wouldn't benefit from caching anymore.